extensions/librdkafka/PublishKafka.h

```diff
@@ -115,14 +121,35 @@ class PublishKafka: public core::Processor {
       buffer.reserve(max_seg_size_);
       read_size_ = 0;
       status_ = 0;
+      rd_kafka_resp_err_t err;
+
+      for (auto kv : flowFile_->getAttributes()) {
+        if (regex_match(kv.first, attributeNameRegex_)) {
+          if (!hdrs) {
+            hdrs = rd_kafka_headers_new(8);
```

Comment:
Was eight randomly chosen? If not please add a comment for any maintainers to understand why you chose this number -- even if any number can be added or removed per  their API.

Reply:
8 was mostly arbitrary, I used the value from https://github.com/edenhill/librdkafka/blob/master/examples/rdkafka_example.c#L416 since some value needed to be used. 0 is also an option but it looks like if 0 is chosen, any additions will grow the list to 16 elements. If a number besides 0 is chosen, when list runs out of room, the list is grown by 2*the list size.